hw_diag/utilities/dbus/modem_manager.py

```diff
@@ -0,0 +1,76 @@
+import dbus
+
+from hw_diag.utilities.dbus.dbus_ids import DBusIds
+from hw_diag.utilities.dbus.dbus_object import DBusObject
+from hw_diag.utilities.dbus.modem import Modem
+from hm_pyhelper.logger import get_logger
+
+logging = get_logger(__name__)
+
+class ModemManager(DBusObject):
+
+    def __init__(self, *args, **kwargs):
+        super(ModemManager, self).__init__(DBusIds.DBUS_MM1_SERVICE,
+                                           DBusIds.DBUS_MM1_PATH,
+                                           DBusIds.DBUS_MM1_IF)
+        self._mm_objmanager_iface = dbus.Interface(self._object_proxy,
+                                                   DBusIds.DBUS_OBJECTMANAGER_IF)
+
+    def all_modems(self):
+        '''
+        @rtype: list of all managed Modem objects
+        '''
+        modem_paths = self._mm_objmanager_iface.GetManagedObjects()
+        modems = [Modem(modem_path) for modem_path in modem_paths]
+        return modems
+
+    def _do_properties_match(self, actual_properties, desired_properties):
+        '''return true if all properties in desired_properties are present
+        and same in actual_properties'''
+        for prop_key in desired_properties:
+            actual_value = actual_properties.get(prop_key)
+            desired_value = desired_properties.get(prop_key)
+            print(f'matching {actual_value} and {desired_value}')
```

Comment:
Use `logging` not `print`, throughout the code.

Reply:
replaced.